runtime/near-evm-runner/Cargo.toml

```diff
@@ -24,8 +24,8 @@ keccak-hash = "0.2.0"
 ripemd160 = "0.9.0"
 libsecp256k1 = "0.3.5"
 
-evm = { git = "https://github.com/paritytech/parity-ethereum", rev = "eed630a002bbebed3a3097127f2483213ff52079" }
-vm = { git = "https://github.com/paritytech/parity-ethereum", rev = "eed630a002bbebed3a3097127f2483213ff52079" }
+evm = { git = "https://github.com/openethereum/openethereum", rev = "v2.6.2" }
+vm = { git = "https://github.com/openethereum/openethereum", rev = "v2.6.2" }
```

Comment:
Do they release those on crates.io at all? :thinking:

Reply:
@frol Doesn't look like: [nothing there for OpenEthereum](https://crates.io/search?q=openethereum), and with the previous name [only a placeholder](https://crates.io/crates/parity-ethereum).